content/documentation/staging/tutorial/05-connect/03-traffic-shifting.adoc

```diff
@@ -0,0 +1,60 @@
+---
+title: Traffic Shifting
+weight: 52
+---
+:icons: font
+:sectlinks:
+
+In the previous link:../tutorial/#01-request-routing[Request Routing] step we have deployed two new versions of the *travels* service using the *travels-v2* and *travels-v3* workloads.
+
+That scenario showed how Istio can route specific requests to specific workloads, in that case, each portal deployed on *travel-portal* namespace (*travels.uk*, *viaggi.it* and *voyages.fr*) were route to a specific *travels* workload (*travels-v1*, *travels-v2* and *travels-v3*).
+
+This Traffic Shifting step will simulate a new scenario: the new *travels-v2* and *travels-v3* workloads will represent new improvements for the *travels* service that will be used by all requests.
+
+These new improvements implemented in *travels-v2* and *travels-v3* represent two alternatives way to address a specific problem and we would like to test them before to decide which one to use as a next version.
+
+So, in a first moment we will send 80% of the traffic into the original *travels-v1* workload, and will split 10% of the traffic on *travels-v2* and *travels-v3*.
+
+NOTE: *[1]* Use the Traffic Shifting Wizard on *travels* service
+
+++++
+<a class="image-popup-fit-height" href="/images/tutorial/05-03-traffic-shifting-action.png" title="Traffic Shifting Action">
+    <img src="/images/tutorial/05-03-traffic-shifting-action.png" style="display:block;margin: 0 auto;" />
+</a>
+++++
+
+{nbsp} +
+Create a scenario with 80% of the traffic distributed on *travels-v1* workload and 10% of the traffic distributed on *travels-v2* and *travels-v3*.
+
+++++
+<a class="image-popup-fit-height" href="/images/tutorial/05-03-split-traffic.png" title="Split Traffic">
+    <img src="/images/tutorial/05-03-split-traffic.png" style="display:block;margin: 0 auto;" />
+</a>
+++++
+
+NOTE: *[2]* Examine Traffic Shifting distribution from the *travels-agency* Graph
+
+++++
+<a class="image-popup-fit-height" href="/images/tutorial/05-03-travels-graph.png" title="Travels Graph">
+    <img src="/images/tutorial/05-03-travels-graph.png" style="display:block;margin: 0 auto;" />
+</a>
+++++
+
+NOTE: *[3]* Compare *travels* workload and study new changes proposed in *travels-v2* and *travels-v3*
+
+Istio Telemetry is grouped per logical application and that gives an advantage to compare different workloads that are related to one or more services that represent a similar concept.
```

Comment:
I found this sentence confusing, I think you are saying something like the following, but this may need more work:

Istio Telemetry is grouped per logical application.  That has the advantage of easily comparing different but related workloads, for one or more services.